packages/nextjs/components/example-ui/ContractData.tsx

```diff
@@ -30,6 +35,19 @@ export const ContractData = () => {
     },
   });
 
+  const {
+    data: events,
+    isLoading: isLoadingEvents,
+    error: errorReadingEvents,
+  } = useScaffoldEventHistory({
+    contractName: "YourContract",
+    eventName: "GreetingChange",
+    fromBlock: Number(process.env.NEXT_PUBLIC_DEPLOY_BLOCK) || 0,
```

Comment:
Maybe we can remove the use of `process.env` and put a straight 0 to keep things consistent?

Reason : 
We currently don't have  any `.env.development` or `.env.production`  

Reason to have them : 
Maybe are we hinting to the user to store the deploy block in `env`  so that they can use same it in another place

Just confused a bit, maybe we keep it as it is. 

cc @sverps , @damianmarti please feel free to merge I think the hook is really looking solid 🔥

Reply:
It was added by @carletex at the hacker houses build.

I think that it's a nice hint to the user to use the deploy block in the env file.

Reply:
Got it makes sense 🙌